draft-wang-ppm-differential-privacy.md

```diff
@@ -348,15 +349,16 @@ presence of the attacker.)
 
 Whether this property holds for a given DP policy depends on which parties can
 be trusted to execute the protocol correctly (i.e., which parties were not
-corrupted by the attacker). The DP policies described in this document are
-secure in one of two trust models described below.
+corrupted by the attacker). We consider three, increasingly pessimistic trust
+models.
 
 ### OAMC: One-Aggregator-most-Clients
 
-Assume that most Clients and one Aggregator is honest. The other
-Aggregator and the Collector are controlled by the atacker. The degree of
-privacy provided by the DP policy (i.e., `EPSILON`) degrades as the number of
-honest Clients decreases.
+Assume that most Clients and one Aggregator is honest and that the other
+Aggregator and the Collector are controlled by the atacker. This corresponds to
+the same trust model as the base DAP procotol. Ideally, the degree of privacy
+provided (i.e., the value of `EPSILON` for pure-DP) degrades gracefully as the
+number of honest Clients decreases.
```

Comment:
Does OAMC really correspond to the DAP trust model? I think that OA corresponds better, and I think that @simon-friedberger made the same point here: https://github.com/wangshan/draft-wang-ppm-differential-privacy/pull/17#discussion_r1329844060. 

If I understand correctly, DAP only says "Individual users can reveal their own measurement and compromise their own privacy." and doesn't limit the number of Clients who do so.

Reply:
The base protocol is meant to provide private aggregation (the attacker learns nothing but the aggregate result) up to Sybil attacks. That means the protocol only achieves its goals when the attacker does not control any of the inputs to aggregation.

I think you're right that it's different, mainly due to the word "most". I've clarified.